package.json

```diff
@@ -2,18 +2,25 @@
   "name": "amn-design-system",
   "private": true,
   "scripts": {
+    "clean": "turbo run clean",
     "build": "turbo run build",
     "dev": "turbo run dev",
-    "check": "biome check --write",
-    "format": "biome format --write",
+    "lint": "biome check --write",
+    "lint:fix": "biome format --write",
     "postinstall": "simple-git-hooks"
   },
   "devDependencies": {
     "@biomejs/biome": "2.2.4",
     "@commitlint/cli": "^20.0.0",
     "@commitlint/config-conventional": "^20.0.0",
+    "@types/react": "^19.1.15",
+    "@types/react-dom": "^19.1.9",
+    "@vitejs/plugin-react": "^5.0.4",
+    "npm-run-all": "^4.1.5",
     "simple-git-hooks": "^2.13.1",
-    "turbo": "^2.5.8"
+    "turbo": "^2.5.8",
+    "typescript": "5.9.2",
+    "vite": "^7.1.7"
```

Comment:
Since this is a personal project and you don't have red tape around using pre-release or RC versions of packages, might I suggest you try using the vite rolldown integration; it is stupidly quicker at compiling/packaging your bundle.

https://vite.dev/guide/rolldown

If you add some apps using NextJS, they don't use vite and so they wont get to use the rolldown integration, but this is fine as NextJS with its turbo shit is very good anyway. Again, something I expect and love.

Of course, if you use TanStack Start, it's built on vite, so it will benefit from rolldown also. 😉